docs/docs/get-started.md

```diff
@@ -6,9 +6,11 @@ icon: material/rocket-launch-outline
 # Getting Started
 
 ## Prerequisites
-Recce requires that your dbt project has two [environments](https://docs.getdbt.com/docs/environments-in-dbt) to compare. For example, one for production and another for development. 
+Recce requires that your dbt project has two separate [environments](https://docs.getdbt.com/docs/environments-in-dbt) that refers to different [schemas](https://docs.getdbt.com/docs/core/connect-data-platform/connection-profiles#understanding-target-schemas). For example, one for production with prod schema and another for development with dev schema.
 
-Prepare two targets with separate [schemas](https://docs.getdbt.com/docs/core/connect-data-platform/connection-profiles#understanding-target-schemas) in your dbt profile. Your `profiles.yml` might look something like this:
+For most data warehouses, Recce can get your schema from your dbt artifacts.
```

Comment:
I'm not sure if this is correct.

Reply:
Not exactly, we may need to change a bit.
We can get the schema from dbt manifest for all data warehouses.
The reason of adding the second target for duckdb is that duckdb is running locally, so users may need to prepare their own production data.